examples/python/easy_start_demo/README.md

```diff
@@ -0,0 +1,28 @@
+## Demo for using gRPC in Python
+
+在Python中使用gRPC时, 进行数据传输的四种方式。(Four ways of data transmission when gRPC is used in Python.)
+
+- #### 简单模式 (unary-unary)
+```text
+没啥好说的,跟调普通方法没差
+There's nothing to say. It's no different from the usual way.
```

Comment:
It's the purpose of this description to describe what "the usual way" is, no? This is a quickstart. Surely there's something to say.